akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -209,6 +210,14 @@ object Source {
   def tick[O](initialDelay: FiniteDuration, interval: FiniteDuration, tick: O): javadsl.Source[O, Cancellable] =
     new Source(scaladsl.Source.tick(initialDelay, interval, tick))
 
+  /**
+   * Same as [[tick]], but accepts [[java.time.Duration]] instead of Scala ones.
+   */
+  def tick[O](initialDelay: Duration, interval: Duration, tick: O): javadsl.Source[O, Cancellable] = {
+    import scala.concurrent.duration._
+    Source.tick(initialDelay.toMillis.millis, interval.toMillis.millis, tick)
```

Comment:
Use: 

```
import akka.util.JavaDurationConverters._
initialDelay.asScala
```

instead maybe?

Reply:
The converter returns Duration instead of FiniteDuration, which is odd, I think we should make it return FiniteDuration, what do you think?